packages/babel-plugin-transform-bigint/test/fixtures/addition/expected.js

```diff
@@ -0,0 +1,2 @@
+import _BigInt from "bigint-polyfill";
```

Comment:
As for the polyfill library, I'd suggest starting with something like [node-bigint](https://github.com/substack/node-bigint). This has all the functions implemented, though it only works on Node, not the web. I don't think this is a problem, as the Babel implementation is more for exploration/demonstration purposes than actual deployment.

On top of whatever base library you start with, there will have to be significant things to match the form of the spec. This includes:
- Dispatching appropriately on Number vs BigInt
- Methods like `asUintN`, which you're unlikely to find elsewhere
- Being a little creative with some functions, for example `BigInt.prototype.valueOf` should throw, to prevent implicit coercion to Number.